lib/internal/Magento/Framework/View/Test/Unit/Element/AbstractBlockTest.php

```diff
@@ -290,9 +290,9 @@ public function getCacheLifetimeDataProvider()
                 'dataFromCache' => 'dataFromCache',
                 'dataForSaveCache' => '',
                 'expectsDispatchEvent' => $this->exactly(2),
-                'expectsCacheLoad' => $this->once(),
+                'expectsCacheLoad' => $this->never(),
                 'expectsCacheSave' => $this->never(),
-                'expectedResult' => 'dataFromCache',
+                'expectedResult' => '',
```

Comment:
Generally looks good. It would better use null for expected result because we started return null in  `lib/internal/Magento/Framework/View/Element/AbstractBlock.php` line: 1077